src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/Et1VettingHelper.java

```diff
@@ -0,0 +1,164 @@
+    private static final String OUTPUT_NAME = "ET1 Vetting.pdf";
+
+    private Et1VettingHelper() {
+        // Access through static methods
+        OBJECT_MAPPER.setSerializationInclusion(JsonInclude.Include.NON_NULL);
+    }
+
+    /**
+     * This method generates the data in a JSON format stored in a String which allows Tornado to process the
+     * information.
+     * @param caseData contains the data from ET1 Vetting
+     * @param userToken contains the authentication token
+     * @return a string which contains the data in JSON Format which Tornado can use to process and generate the
+     document
+     * @throws JsonProcessingException if the JSON cannot be generated correctly, an error would be thrown. This could
+     be due to an illegal character potentially existing in the data
+     */
+    public static String getDocumentRequest(CaseData caseData, String userToken) throws JsonProcessingException {
+        Et1VettingData et1VettingData = Et1VettingData.builder()
+                .ethosCaseReference(caseData.getEthosCaseReference())
+                .et1VettingCanServeClaimYesOrNo(nullChecker(caseData.getEt1VettingCanServeClaimYesOrNo()))
+                .et1VettingCanServeClaimNoReason(nullChecker(caseData.getEt1VettingCanServeClaimNoReason()))
+                .et1VettingCanServeClaimGeneralNote(nullChecker(caseData.getEt1VettingCanServeClaimGeneralNote()))
+                .et1VettingAcasCertIsYesOrNo1(nullChecker(caseData.getEt1VettingAcasCertIsYesOrNo1()))
+                .et1VettingAcasCertExemptYesOrNo1(nullChecker(caseData.getEt1VettingAcasCertExemptYesOrNo1()))
+                .et1VettingAcasCertIsYesOrNo2(nullChecker(caseData.getEt1VettingAcasCertIsYesOrNo2()))
+                .et1VettingAcasCertExemptYesOrNo2(nullChecker(caseData.getEt1VettingAcasCertExemptYesOrNo2()))
+                .et1VettingAcasCertIsYesOrNo3(nullChecker(caseData.getEt1VettingAcasCertIsYesOrNo3()))
+                .et1VettingAcasCertExemptYesOrNo3(nullChecker(caseData.getEt1VettingAcasCertExemptYesOrNo3()))
+                .et1VettingAcasCertIsYesOrNo4(nullChecker(caseData.getEt1VettingAcasCertIsYesOrNo4()))
+                .et1VettingAcasCertExemptYesOrNo4(nullChecker(caseData.getEt1VettingAcasCertExemptYesOrNo4()))
+                .et1VettingAcasCertIsYesOrNo5(nullChecker(caseData.getEt1VettingAcasCertIsYesOrNo5()))
+                .et1VettingAcasCertExemptYesOrNo5(nullChecker(caseData.getEt1VettingAcasCertExemptYesOrNo5()))
+                .et1VettingAcasCertIsYesOrNo6(nullChecker(caseData.getEt1VettingAcasCertIsYesOrNo6()))
+                .et1VettingAcasCertExemptYesOrNo6(nullChecker(caseData.getEt1VettingAcasCertExemptYesOrNo6()))
+                .et1VettingAcasCertGeneralNote(nullChecker(caseData.getEt1VettingAcasCertGeneralNote()))
+                .substantiveDefectsList(CollectionUtils.isEmpty(caseData.getSubstantiveDefectsList())
+                        ? null
```

Comment:
Have you tried making a utility method for this isEmpty() ? null : .toString() pattern?

Reply:
Utility method created